### PR TITLE
feat(which-key): added search command for colour highlights

### PR DESCRIPTION
**Pull request by danielo515**
# Description

This adds a new search section on the default which key config to search for colour highlights.
This is a builtin telescope feature, so no new dependencies were added.
I think this is a cool addition because it enhances the discoverability of something that is not that commonly exposed 
to vim users, and will help a lot in customising the theme to the users likings.

## How Has This Been Tested?

- `<leader>sH`
- See the picker for highlight with colour preview
- ...

This is how it looks like
image

### CODE REVIEW

**Review by MikiVanousek — APPROVED** [review]

File: lua/lvim/core/which-key.lua [hunk 1]
```diff
@@ -244,6 +244,7 @@ M.config = function()
         c = { "<cmd>Telescope colorscheme<cr>", "Colorscheme" },
         f = { "<cmd>Telescope find_files<cr>", "Find File" },
         h = { "<cmd>Telescope help_tags<cr>", "Find Help" },
+        H = { "<cmd>Telescope highlights<cr>", "Find color highlights" },
```
Comment (kylo252) [inline lua/lvim/core/which-key.lua:247]:
```suggestion
        H = { "<cmd>Telescope highlights<cr>", "Find highlight groups" },
```

**Review by kylo252 — COMMENTED** [review]
Sorry for the delay! 

just a small modification, but otherwise LGTM! 🚀

**Comment by danielo515** [comment]
Done!

**Comment by kylo252** [comment]
please rebase on latest rolling, it should solve the issues with the CI.

**Comment by danielo515** [comment]
@kylo252 done!

> please rebase on latest rolling, it should solve the issues with the CI.

**Review by kylo252 — APPROVED** [review]

**Pull request merged**